enclave_build/Cargo.toml

```diff
@@ -21,4 +21,5 @@ url = "2.1"
 sha2 = "0.9.5"
 futures = "0.3.13"
 
+oci-distribution = "0.9.2"
```

Comment:
CI build fails with
```
 Compiling oci-distribution v0.9.2
error[E0658]: destructuring assignments are unstable
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/oci-distribution-0.9.2/src/client.rs:379:31
    |
379 |             (location, start) = self.push_chunk(&location, image, blob_data, start).await?;
    |             ----------------- ^
    |             |
    |             cannot assign to this expression
    |
    = note: see issue #71126 <https://github.com/rust-lang/rust/issues/71126> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `oci-distribution` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
make: *** [.build-nitro-cli] Error 101 Compiling oci-distribution v0.9.2
error[E0658]: destructuring assignments are unstable
   --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/oci-distribution-0.9.2/src/client.rs:379:31
    |
379 |             (location, start) = self.push_chunk(&location, image, blob_data, start).await?;
    |             ----------------- ^
    |             |
    |             cannot assign to this expression
    |
    = note: see issue #71126 <https://github.com/rust-lang/rust/issues/71126> for more information

For more information about this error, try `rustc --explain E0658`.
error: could not compile `oci-distribution` due to previous error
warning: build failed, waiting for other jobs to finish...
error: build failed
make: *** [.build-nitro-cli] Error 101
```

it looks like they built the crate with nightly.
we build with stable.
maybe try to find an older version that works with stable